omod-2.0/src/test/java/org/openmrs/module/attachments/obs/TestAttachmentBytesViewHandler.java

```diff
@@ -7,43 +7,39 @@
 import org.openmrs.obs.handler.BinaryDataHandler;
 
 public class TestAttachmentBytesViewHandler extends AbstractAttachmentHandler {
-	
+
 	public static final String DEFAULT_VIEW = "VIEW_1";
-	
+
 	public static final String DEFAULT_VIEW_DATA = "This is a string content for VIEW_1.";
-	
+
 	public static final String ALTERNATE_VIEW = "VIEW_2";
-	
+
```

Comment:
@sherrif10   thanks for taking up this you could probably also check your  eclipse formatter,it could be the cause of all these spaces

Reply:
Sure thanks @herbertYiga , will do that